javapackages-tools.spec

```diff
@@ -11,6 +11,8 @@
 %global pkg_name javapackages-tools
 %{?scl:%scl_package javapackages-tools}
 
+%global rpmmacrodir %{?scl:%{_root_sysconfdir}/rpm}%{!?scl:%{_prefix}/lib/rpm/macros.d}
```

Comment:
1. Value of rpmmacrodir should not be SCL-dependant
2. rpmmacrodir is already defined in Fedora (as /usr/lib/rpm/macros.d), so it should be re-defined only in RHEL (as /etc/rpm)